source/read.txt

```diff
@@ -16,3 +30,134 @@ Read Data from MongoDB
    /read/specify-a-query
    /read/distinct
    /read/change-streams
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+{+driver-short+} methods for retrieving documents.
+
+.. tip::
+
+   To learn more about any of the methods shown on this page, see the link
+   provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <php-read-sample>` or your own application.
+Be sure to replace all placeholders, such as ``<connection string>``, with
```

Comment:
The `<connection string>` placeholder doesn't actually appear in the code sample. Suggest either adding it in there or changing this for consistency.